frontends/ol-components/src/components/TruncateText/TruncateText.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { TruncateText } from "./TruncateText"
+
+const meta: Meta<typeof TruncateText> = {
+  title: "ol-components/TruncateText",
+  component: TruncateText,
+  argTypes: {
+    lineClamp: {
+      options: ["none", 1, 2, 3, 4, 5],
+      control: {
+        type: "select",
+      },
+    },
+  },
+  args: {
+    lineClamp: 1,
+    children:
+      "This is a long text that should be truncated after a few lines. ".repeat(
+        30,
+      ),
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof TruncateText>
+
+export const None: Story = {
+  args: {
+    lineClamp: "none",
```

Comment:
A bit of a nitpick, but the "none" example here in Storybook still says "This is a long text that should be truncated after a few lines" when it fact because `lineClamp` is set to "none," it should not be truncated.